cfn-resources/util/constants/constants.go

```diff
@@ -99,4 +99,10 @@ const (
 	AppID       = "AppId"
 	RealmPubKey = "RealmConfig.PublicKey"
 	RealmPvtKey = "RealmConfig.PrivateKey"
+
+	UserName             = "UserName"
+	ExportBucketID       = "ExportBucketId"
+	ExportID             = "ExportId"
+	ErrorExportJobCreate = "error creating Export Job for the project(%s) : %s"
```

Comment:
Error constants should be at resource level since they are not common across resources, can you move to resource level?

Reply:
fixed